ml4bio/ml4bio.py

```diff
@@ -2222,17 +2241,47 @@ def initUI(self):
         self.rocRadioButton.toggled.connect(self.plot)
         self.prRadioButton.toggled.connect(self.plot)
 
-        # global geometry
-        self.resize(1064, 700)
         self.setWindowTitle('ML4Bio (version {})'.format(ml4bio.__version__))
-        self.leftPanel.resize(360, 680)
-        self.leftPanel.setStyleSheet("QStackedWidget {background-color:rgb(226, 226, 226)}")
-        self.leftPanel.move(10, 10)
-        self.rightPanel.resize(680, 680)
-        self.rightPanel.move(380, 10)
+
+        self.leftPanel.resize(0.34*w, h*0.95)
+        self.leftPanel.move(0.01*w, 0.02*h)
+        self.rightPanel.resize(0.60*w, 0.95*h)
+        self.rightPanel.move(0.36*w, 0.02*h)
 
         self.show()
 
+
+    def resizeEvent(self, event):
+        """
+        Catches the resize event to set things up
+        """
+        w = self.width()
+        h = self.height()
+        self.leftPanel.resize(0.34*w, h*0.95)
+        self.leftPanel.move(0.01*w, 0.02*h)
+        self.rightPanel.resize(0.60*w, 0.95*h)
+        self.rightPanel.move(0.36*w, 0.02*h)
+
```

Comment:
Is it possible to scale the hyperparameter list on the second page when one resizes the GUI? Currently some text overlaps if the GUI is not tall enough.

Reply:
I figured it out. It was actually because we set the minimum size of the hyperparameter list, so when it hits the min it just starts overlapping other layouts. I removed "self.paramStack.setMinimumHeight(320)" and there doesn't appear to be an issue anymore.